bin/i18n/i18n_script_utils.rb

```diff
@@ -16,8 +16,8 @@ class I18nScriptUtils
   CROWDIN_CREDS_PATH = CDO.dir('bin/i18n/crowdin_credentials.yml').freeze
   PROGRESS_BAR_FORMAT = '%t: |%B| %p% %a'.freeze
   PARALLEL_PROCESSES = Parallel.processor_count.freeze
-  SOURCE_LOCALE = 'en-US'.freeze
-  TTS_LOCALES = (::TextToSpeech::VOICES.keys - %i[en-US]).freeze
+  SOURCE_LOCALE = I18n.default_locale.to_s.freeze
+  TTS_LOCALES = (::TextToSpeech::VOICES.keys - [SOURCE_LOCALE.to_i]).freeze
```

Comment:
should it be `to_sym` instead of `to_i`? We could also use `%I[#{SOURCE_LOCALE}]`